tests/test_state_metrics/test_fidelity_of_separability.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for State Fidelity of Seperability."""
+
+import numpy as np
+import pytest
+from toqito.states import bell
+from toqito.state_metrics import fidelity_of_separability
+from toqito.matrix_ops import tensor
+
+
+# test mixed rho
+mixed_rho = 1/2 * bell(0) * bell(0).conj().T + 1 / 2 * bell(
```

Comment:
For vector multiplication, I think we should prefer using the `@` operator over the `*` operator.